### ISSUE TITLE
Add labels to form fields

### ISSUE THREAD

**Issue by mredar** [ucldc/avram#41]
AMP flags lack of labels as a severity HIGH problem

https://uc.ssbbartgroup.com/public/reporting/view_module.php?module_id=2615622 
(is the report above viewable by not logged in users?)

Need to add labels such as:

<label for="name">Collection Title</label>
<input name="name" placeholder="Collection Title" class="span11 title-input" type="text">
